docs/contributing/tests.md

```diff
@@ -50,19 +50,29 @@ samples are provided. To run the tests, follow this procedure:
 
   ```
   sudo dnf -y install python-devel
-  python3 -m venv venv
-  source venv/bin/activate
-  pip install openstackclient osc_placement jmespath
+  pip install --user openstackclient osc_placement jmespath
   ansible-galaxy collection install community.general
   ```
 
+* (on Centos9 Stream) Apply `jmespath` workarounds for ansible-core 2.15 and ansible 8.5:
+  ```
+  pip install --user ansible yq
+  sudo su
```

Comment:
Is sudo really necessary here?

Reply:
yes, it didn't work for non root user shell